charts/k8s-monitoring/test/query-test.sh

```diff
@@ -164,6 +180,11 @@ for i in $(seq 0 "${count}"); do
         exit 1
       fi
       ;;
+    profiles)
```

Comment:
Does Pyroscope have a name for its query language? `ProfileQL`? `PyroQL`? The literal 🔥 emoji?

Reply:
So far we don't have a language other than the label selectors that we handle like LogQL/PromQL, but I think `profileql` or `pyroql` is good enough for those purposes. We can obvisouly also go on a UTF-8 journey with the emoji 😆